arroyo-api/src/sources.rs

```diff
@@ -196,6 +196,21 @@ pub enum PrimitiveType {
     UnixMillis,
 }
 
+fn primitive_to_sql(primitive_type: &PrimitiveType) -> &'static str {
+    match primitive_type {
+        PrimitiveType::Int32 => "INTEGER",
+        PrimitiveType::Int64 => "BIGINT",
+        PrimitiveType::UInt32 => "UNSIGNED INTEGER", // not really sql
+        PrimitiveType::UInt64 => "UNSIGNED BIGINT",  // not really sql
+        PrimitiveType::F32 => "FLOAT",
+        PrimitiveType::F64 => "DOUBLE",
+        PrimitiveType::Bool => "BOOLEAN",
+        PrimitiveType::String => "VARCHAR",
+        PrimitiveType::Bytes => "BLOB",
```

Comment:
The datafusion page calls this `BYTEA`, but we can kinda pick. Could also go with BINARY

Reply:
I'd go with BINARY or BYTEA over BLOB. BLOB is "binary large object" and is used for storing files in the DB (like images). BYTEA/BINARY and used for smaller binary strings.

Reply:
Done, made it `BINARY`